test/ref_ops_test.cpp

```diff
@@ -5138,6 +5139,68 @@ TEST_CASE(nms_test)
     EXPECT(migraphx::verify_range(result, gold));
 }
 
+/*
```

Comment:
why is it commented out ?

Reply:
I think it was breaking earlier. Let me fix this.